text/0013-schema-enums.md

```diff
@@ -0,0 +1,145 @@
+# Enums in Schemas
+
+## Related issues and PRs
+
+- Reference Issues: 
+- Implementation PR(s): 
+
+## Timeline
+
+- Start Date: 2023-06-30
+- Date Entered FCP:
+- Date Accepted:
+- Date Landed:
+
+## Summary
+
+Add support for enumerations of `bool`, `string`, and `long` types in schemas, and use this information when doing policy validation and schema-based-parsing.
```

Comment:
I'm not fully convinced we should add long enums at the same time as we have long ranges PR is progress. We should think a bit more carefully about how they behave when in the language together.

Reply:
I agree this warrants more conversation.